specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchindex.json

```diff
@@ -245,6 +251,93 @@
               "name": "SearchOptions"
             }
           },
+          {
+            "name": "queryLanguage",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "none",
+              "en-us"
+            ],
+            "x-ms-enum": {
+              "name": "QueryLanguage",
+              "modelAsString": false,
+              "values": [
+                {
+                  "value": "none",
+                  "name": "None",
+                  "description": "Query language not specified."
+                },
+                {
+                  "value": "en-us",
+                  "name": "EnUs",
+                  "description": "English"
+                }
+              ]
+            },
+            "description": "The language of the query.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "speller",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "none",
+              "lexicon"
+            ],
+            "x-ms-enum": {
+              "name": "Speller",
+              "modelAsString": false,
+              "values": [
+                {
+                  "value": "none",
+                  "name": "None",
+                  "description": "Speller not enabled."
+                },
+                {
+                  "value": "lexicon",
+                  "name": "Lexicon",
+                  "description": "Speller corrects individual query terms using a static lexicon for the language specified by the queryLanguage parameter."
+                }
+              ]
+            },
+            "description": "Improve search recall by spell-correcting individual search query terms.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "answers",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "none",
+              "extractive"
+            ],
+            "x-ms-enum": {
+              "name": "Answers",
+              "modelAsString": false,
```

Comment:
Same here, if we want customer to be able to pass |count-X, we might need to model this as a string